File: docs/data-tools/codesnippet/VisualBasic/query-datasets_2.vb
        Dim s As String = "primaryKeyValue"
        Dim foundRow As DataRow = DataSet1.Tables("AnyTable").Rows.Find(s)

        If foundRow IsNot Nothing Then
            MsgBox(foundRow(1).ToString())
        Else
            MsgBox("A row with the primary key of " & s & " could not be found")
        End If

File: docs/data-tools/codesnippet/CSharp/query-datasets_2.cs
            string s = "primaryKeyValue";
            DataRow foundRow = dataSet1.Tables["AnyTable"].Rows.Find(s);

            if (foundRow != null) 
            {
                MessageBox.Show(foundRow[0].ToString());
            }
            else
            {
                MessageBox.Show("A row with the primary key of " + s + " could not be found");
            }